beater/handlers.go

```diff
@@ -176,13 +176,17 @@ func logHandler(h http.Handler) http.Handler {
 
 		h.ServeHTTP(lw, lr)
 
-		reqLogger.Infow("handled request", "response_code", lw.Code,
-			"method", r.Method, "URL", r.URL, "content_length", r.ContentLength,
-			"remote_address", utility.ExtractIP(r), "user-agent", r.Header.Get("User-Agent"))
-
 		if lw.Code > 399 {
 			responseErrors.Inc()
 		} else {
+			reqLogger.Infow("handled request", []interface{}{
```

Comment:
cool! What do you think about moving the parameters here (`response_code`, `method` etc.) up to when we're creating the `reqLogger`: 

```
reqLogger := logger.With(
    "request_id", reqID
    "response_code", lw.Code,
    "method", r.Method,
    "URL", r.URL,
    "content_length", r.ContentLength,
    "remote_address", utility.ExtractIP(r),
    "user-agent", r.Header.Get("User-Agent"),
)
```

this will allow you to remove it from `sendStatus`